docs/rst/introduction/tools/shapes_demo.rst

```diff
@@ -3,5 +3,136 @@
 ROS 2 Shapes Demo
 =================
 
+*ROS 2 Shapes Demo* is an application in which Publishers and Subscribers are shapes of different colors and sizes moving on a board.
+Each shape refers to its own topic: Square, Triangle or Circle.
+A single instance of the *ROS 2 Shapes Demo* can publish on or subscribe to several topics at a time.
+
+It demonstrates the capabilities of eProsima Fast DDS or as a proof of interoperability with other RTPS-compliant implementations.
+
+.. raw:: html
+
+    <video width=100% height=auto autoplay loop controls>
+        <source src="../../../_static/interoperability.mp4">
+        Your browser does not support the video tag.
+    </video>
+    <br></br>
+
+Using ROS 2 Shapes Demo
+--------------------------
+
+After the executable is launched, a window similar to the one presented in the following image should be displayed.
+
+.. figure:: /rst/figures/intro/tools/shapes_demo/mainWindow.png
+   :alt: Main Window
+   :align: center
+
+
+Publishing a Topic
+^^^^^^^^^^^^^^^^^^
+
+The Publish button allow the users to define the Shape (topic) and Quality of Service (QoS) for their publication.
+The following image shows an example of the Publication menu.
+
+.. figure:: /rst/figures/intro/tools/shapes_demo/publish.png
+   :scale: 100 %
+   :alt: Publish Window
+   :align: center
+
+There are multiple parameters that the user can define in this menu:
+
+*   Shape: This parameter defines the topic where the publication is going to occur.
+    Three different shapes can be published: ``Square``, ``Circle`` and ``Triangle`` (see Fast DDS Topic Documentation).
+*   Color: The user can define the color of the shape.
```

Comment:
Add a note that ROS2 does not have keys?

Reply:
I've removed all mentions to topics with keys instead.